Revise census metrics to reflect current reality (#547)
mozilla/firefox-data-docs: src/concepts/censuses.md

> ## Legacy Census Metrics

Reviewer: Should this section also include the definitions for aDAU and aMAU?

Reviewer 2: fwiw there's some internal commentary at https://mozilla-private.report/smoot-existing-metrics/book/20_metric_correlations.html#dau-and-dau-5-uris; it's probably good to have an aDAU grave marker on DTMO and I'll toss that on my todo list.

Author: Yeah, moving https://docs.telemetry.mozilla.org/cookbooks/active_dau.html to the historical section is also on my list of things to do (of course I wouldn't object if @tdsmith did it for me!)

### DAU / Daily Active Users

> > **⚠** The mechanism described below is no longer used: see the [DAU definition in metrics](../metrics/metrics.md#daily-active-users-dau) for the currently-accepted definition.

Reviewer 2: I don't think this has changed.

Author: There's overlap between this and the definition in metrics for sure, but I think we need to make it clear that this isn't the currently accepted one. I'm open to changing the wording of this warning, but I think we should maintain it.

Reviewer 2: ```suggestion
> **⚠** This description of DAU is not authoritative; please see the [DAU definition in metrics](../metrics/metrics.md#daily-active-users-dau) for the canonical definition.
```

> > **⚠** The mechanism described below is no longer used: see the [DAU definition in metrics](../metrics/metrics.md#daily-active-users-dau) for the currently-accepted definition.

Author: How about:

```suggestion
> **⚠** This is out of date. See the [DAU definition in metrics](../metrics/metrics.md#daily-active-users-dau) for the currently-accepted version.
```
